packages/main/src/MonthPicker.ts

```diff
@@ -36,7 +36,7 @@ import MonthPickerTemplate from "./generated/templates/MonthPickerTemplate.lit.j
 import monthPickerStyles from "./generated/themes/MonthPicker.css.js";
 
 const PAGE_SIZE = 12; // total months on a single page
-const ROW_SIZE = 3; // months per row (4 rows of 3 months each)
+let ROW_SIZE = 3; // months per row (4 rows of 3 months each)
```

Comment:
lets make this variable per instance otherwise it might result in wrong visual appearance